docs/installation.rst

```diff
@@ -51,18 +51,22 @@ URL paths
 ~~~~~~~~~
 
 Include the required CAS URL endpoints in your project's root ``URLconf``
-with the following::
+in ``urls.py`` with the following::
```

Comment:
Hi @jaller94, it would be great if you could rephase "in ``urls.py``" to something like "(``usually urls.py``)"

Reply:
```suggestion
(usually ``urls.py``) with the following::
```